.github/ISSUE_TEMPLATE/bug_report.yaml

```
@@ -0,0 +1,67 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: bug-description
+    attributes:
+      label: Describe the bug
+      description: Provide a general description of the bug, in plain text. Use the dedicated fields below for code snippets.
+      placeholder: Description of the bug
+    validations:
+      required: true
+  - type: textarea
+    id: java-input
+    attributes:
+      label: Source code you are trying to analyze/transform
+      description: Provide a minimal example of source code you are trying to analyze/transform that causes the bug to appear.
+      placeholder: System.out.println("Hello, world!");
+      render: Java
+    validations:
+      required: false
+  - type: textarea
+    id: spoon-code
+    attributes:
+      label: Source code for your Spoon processing
+      description: Provide your processing code that uses Spoon and causes the bug to appear. This will be automatically formatted, so no need for backticks.
+      placeholder: System.out.println("Hello, world!");
+      render: Java
+    validations:
+      required: false
+  - type: textarea
+    id: output
+    attributes:
+      label: Output
```

Maybe make the distinction between expected and actual a bit clearer? `Your`/`Actual`/`Observed`/... I don't know.
```suggestion
      label: Actual output
```

---

Agreed, that's a good distinction.